### ISSUE TITLE
Cannot import big slashing protection

### ISSUE THREAD

**Issue by caevv** [ChainSafe/lodestar#5392]
`Internal server error. Error: Maximum call stack size exceeded`

Running with `--keymanager.bodyLimit 11125829120`

`28M` file

**Comment by nflaig** [comment]
Thanks for reporting @caevv, I will look into this

**Comment by nflaig** [comment]
@caevv I am not able to reproduce the issue, even with a bigger slashing protection file it seems to work fine for me. Would it be possible for you to provide some logs of your validator client? The error log should contain the stack trace which would be really helpful to further analyse this issue.

You can observe the logs by running `./ethd logs -f validator` or you could just search for the specific log by running `./ethd logs validator | grep "Maximum call stack size exceeded" -A10 -B10`.

**Comment by caevv** [comment]
Sorry for the delay, here it is:
```
eth-docker-validator-1  | May-03 19:45:18.833[]                 info: Verified connected beacon node and validator have the same genesisValidatorRoot
eth-docker-validator-1  | May-03 19:45:18.835[]                 info: doppelganger protection enabled detectionEpochs=1
eth-docker-validator-1  | May-03 19:45:18.872[]                 info: Started REST API server address=http://0.0.0.0:7500
eth-docker-validator-1  | May-03 19:45:18.872[]                 warn: REST API server is exposed, ensure untrusted traffic cannot reach this API
eth-docker-validator-1  | May-03 19:45:18.872[]                 info: REST api server keymanager bearer access token located at:
eth-docker-validator-1  |
eth-docker-validator-1  | /var/lib/lodestar/validators/validator-db/api-token.txt
eth-docker-validator-1  |
eth-docker-validator-1  | May-03 19:57:53.054[]                error: Req req-1 importKeystores error  Maximum call stack size exceeded
eth-docker-validator-1  | RangeError: Maximum call stack size exceeded
eth-docker-validator-1  |     at minEpoch (file:///usr/app/packages/validator/src/slashingProtection/utils.ts:31:35)
eth-docker-validator-1  |     at SlashingProtectionAttestationService.importAttestations (file:///usr/app/packages/validator/src/slashingProtection/attestation/index.ts:148:28)
eth-docker-validator-1  |     at processTicksAndRejections (node:internal/process/task_queues:95:5)
eth-docker-validator-1  |     at SlashingProtection.importInterchange (file:///usr/app/packages/validator/src/slashingProtection/index.ts:64:7)
eth-docker-validator-1  |     at KeymanagerApi.importKeystores (file:///usr/app/packages/cli/src/cmds/validator/keymanager/impl.ts:123:7)
eth-docker-validator-1  |     at Object.handler (file:///usr/app/packages/api/src/utils/server/genericJsonServer.ts:44:23)
```

```
❯ ls -la ~/eth-docker/.eth/validator_keys/
total 28544
drwxr-xr-x 2 a a       75 May  3 19:44 .
drwxr-xr-x 4 a a       58 Apr 19 21:17 ..
-rw-r--r-- 1 a a      786 May  3 19:44 keystore-0.json
-rwxr-xr-- 1 a a 29223063 May  3 19:44 slashing_protection.json
```

```
❯ cat .env
...
VC_EXTRAS='--keymanager.bodyLimit 11125829120'
...
```

**Comment by nflaig** [comment]
Thanks @caevv for providing the logs, that helped a lot, a fix is on the way!

**Comment by nflaig** [comment]
@caevv The issue should be fixed in [chainsafe/lodestar:next](https://hub.docker.com/layers/chainsafe/lodestar/next/images/sha256-da0d453345105343bc44062ded1aca7b796c8f89f934dced624263be11022b3a?context=explore) release and will be included in v1.9.0 which will be released in ~2-3 weeks

Using eth-docker you can test if it works by updating [LS_DOCKER_TAG](https://github.com/eth-educators/eth-docker/blob/3e5fd829f4be623ce30cbd8cf4048a2d353b0509/default.env#L184) in your `.env` file

```
LS_DOCKER_TAG=next
```
